packages/twenty-front/src/modules/activities/calendar/components/Calendar.tsx

```diff
@@ -17,19 +37,109 @@ const StyledContainer = styled.div`
   gap: ${({ theme }) => theme.spacing(8)};
   padding: ${({ theme }) => theme.spacing(6)};
   width: 100%;
+  overflow: scroll;
 `;
 
 const StyledYear = styled.span`
   color: ${({ theme }) => theme.font.color.light};
 `;
 
-export const Calendar = () => {
-  const { records: calendarEvents } = useFindManyRecords<CalendarEvent>({
-    objectNameSingular: CoreObjectNameSingular.CalendarEvent,
-    orderBy: { startsAt: 'DescNullsLast', endsAt: 'DescNullsLast' },
-    useRecordsWithoutConnection: true,
+export const Calendar = ({ entity }: { entity: ActivityTargetableObject }) => {
```

Comment:
I now realize that we wanted to avoid the name entity. `activityTargetableObject` looks more explicit to me. Do you mind updating it in `EmailThreads` as well?